4.7/Sub:add rc details (#6850)
ArduPilot/ardupilot_wiki: sub/source/docs/rc-setup.rst

.. _rc-setup:

=============
Radio Control
=============

> Due to the RF attenuation of even a short distance of water above a submersible, radio control must be accomplished by floating tethered bouy to the vehicle which contains the RF components and antenna. This applies also to RF based telemetry, whether independent of, or integrated into, the radio control system.

Reviewer: Possibly worth including [an explanatory link](https://en.wikipedia.org/wiki/Electromagnetic_absorption_by_water#Microwaves_and_radio_waves), for the curious? :-)

Author: TMI I think....they can explore if they want the "why"

To enable radio control, the :ref:`RC_PROTOCOLS<RC_PROTOCOLS>` parameter must be set to a non-zero value, "1" is usually used. If rc control of the flight mode is desired, in addition to axis/directional control, the :ref:`FLTMODE_CH<FLTMODE_CH>` should be set to the RC channel that will control flight modes, if this is desired.

> Any ArduPilot compatible :ref:`radio control system<common-rc-systems>` can be employed.

Reviewer: ```suggestion
Any ArduPilot-compatible :ref:`radio control system<common-rc-systems>` can be employed.
```

Author: Grammarly says non hyphened is correct

Configuration
=============

> The mapping of which incoming rc channel controls which axis/directional control is fixed when GCS only control is used, However, these can be remapped using:

Reviewer: ```suggestion
The mapping of which incoming RC channel controls which axis/directional control is fixed when GCS only control is used, however, these can be remapped using:
```
Is it actually fixed? It used to be, but if RCMAP has been re-enabled then I believe that applies to both RC and `RC_CHANNELS_OVERRIDE`.

Author: when my PR is merged RCMAP will work all the time

- :ref:`RCMAP_ROLL<RCMAP_ROLL>`
- :ref:`RCMAP_PITCHL<RCMAP_PITCH>`
- :ref:`RCMAP_YAW<RCMAP_YAW>`
- :ref:`RCMAP_THROTTLE<RCMAP_THROTTLE>`
- :ref:`RCMAP_FORWARD<RCMAP_FORWARD>`
- :ref:`RCMAP_LATERAL<RCMAP_LATERAL>`

> If a flight mode switch is used, it can be setup as follows:

Reviewer: ```suggestion
If a flight mode switch is used, it can be set up as follows:
```
